src/components/Selectors/CheckBox/CheckBoxGroup.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from 'react';
+import { CheckBoxProps } from '../index';
+import { SelectorSize } from '../Selectors.types';
+import { CheckBox } from './CheckBox';
+
+export const CheckBoxGroup: FC<CheckBoxProps> = ({
+    inputType = 'checkbox',
+    items,
```

Comment:
CheckBoxGroup should have an onChange event, managing invidually for each checkbox will become cumbersome